packages/builders/src/interactions/slashCommands/SlashCommandBuilder.ts

```diff
@@ -74,12 +84,48 @@ export class SlashCommandBuilder {
 	 * @param value Whether or not to enable this command by default
 	 *
 	 * @see https://discord.com/developers/docs/interactions/application-commands#permissions
+	 * @deprecated Use `setDefaultMemberPermissions` and `setDMPermission` instead.
```

Comment:
```suggestion
	 * @deprecated Use `setDefaultMemberPermissions` or `setDMPermission` instead.
```
consistency with the other `@deprecated` tags